.github/workflows/workflow.yml

```diff
@@ -110,3 +110,18 @@ jobs:
       - name: Build configurator
         run: opam install ./dune-configurator.opam
         if: ${{ matrix.configurator == true }}
+
+  lint-fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Use OCaml 4.14.x
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: 4.14.x
+          dune-cache: true
+      - name: Install deps on Unix
+        run: opam install ocamlformat.$(awk -F = '$$1 == "version" {print $$2}' .ocamlformat)
```

Comment:
Can't we just have a makefile target such as `make install-ocamlformat` that does this? I don't like embedding sh into yaml